plugins/uefi-capsule/fu-uefi-capsule-device.c

```diff
@@ -34,6 +34,7 @@ G_DEFINE_TYPE_WITH_PRIVATE(FuUefiCapsuleDevice, fu_uefi_capsule_device, FU_TYPE_
 #define GET_PRIVATE(o) (fu_uefi_capsule_device_get_instance_private(o))
 
 #define FU_EFI_FMP_CAPSULE_GUID "6dcbd5ed-e82d-4c44-bda1-7194199ad92a"
+#define FU_EFI_AB_INDICATIONS_GUID "4a8dd2d2-8acf-11ef-b864-0242ac120002"
```

Comment:
@apalos what's this GUID please? Got a better name?

Reply:
the name seems fine. We already define GUIDs for accept/revert capsules here https://mbfw.readthedocs.io/en/latest/chapter2-uefi.html#uefi-extra-dependencies, 

I can reuse the GUID you propose for the variables when I update the spec

Reply:
Ohh I stole the GUID from https://gitlab.com/Linaro/trustedsubstrate/mbfw/-/merge_requests/6/diffs -- it probably ought to be something more official, no?

Reply:
Oh you are right -- I completely forgot about that PR. That's the one I plan on moving over to the new repo Arm is hosting, so that's the GUID we'll eventually use